MetaMask/metamask-extension
# Fixing 'View on Opensea' link for main and testnet NFTs

Fixes: https://github.com/MetaMask/metamask-extension/issues/19782

## Explanation
This updates the generated NFT detail link to accommodate OpenSea's link scheming for Ethereum mainnet, Goerli, and Sepolia. 

### Before
https://github.com/MetaMask/metamask-extension/assets/8732757/9d08f477-40e9-4278-8cfa-7af66721902a

### After
https://github.com/MetaMask/metamask-extension/assets/8732757/d24309c7-b6b1-401b-982c-d197d9b3b0ff

## Manual Testing Steps
1. Mint/Add NFTs to a wallet for the main, goerli, and sepolia networks
2. For each network's NFT's respectively, click through to the NFT detail page.
3. In the NFT options, click on `View on Opensea`
4. Ensure that you are taken to the valid Opensea page for that NFT

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
